public/js/index.js

```diff
@@ -594,6 +594,7 @@ function initWikiForm() {
                         function (data) {
                             preview.innerHTML = '<div class="markdown">' + data + '</div>';
                             emojify.run($('.editor-preview')[0]);
+                            $('.editor-preview').autolink();
```

Comment:
@lunny yes, as server-side autolinking often pulled apart tags, attributes. On client side we can manipulate DOM the way we want to, with no harm done to structure.
@tboerger about search engine indexing.
Default ```<http://google.com>``` markdown syntax is always done on server side. Autolinking in plain text is not defined by any markdown standard, so it's just nice stuff to have. If we do plain text autolinking on server, we can corrupt the whole document. Wrong thing to do, so I disabled it. If we do it on client, we may not linkify all possible links, but we do it in much safer way for the document. So, why not. Even if user has disabled JS in his browser, standard links will still work, plain text will not. Better than rendering corrupted document either way. Plus, most popular search engines (like Google, Bing, Yandex) execute JavaScript, so no harm done.